### PR TITLE
Make charter obsolescence clearer

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

File: src/charter [hunk 1]
```diff
@@ -16,16 +16,15 @@
...
 
-  <p>Note: This document was written in 2004 and no longer represents anything like how the WHATWG operates. For a more up to date
-  discussion on how the WHATWG functions today, the <a href="/faq">FAQ</a> does a better job. The original charter is included below
-  to provide historical context.</p>
-
-  <p>See also: <a href="charter.pl">Polish translation of the charter in April 2007</a></p>
+  <p><strong>Note: This document was written in 2004 and no longer represents how the WHATWG
+  operates. For a more up to date description read <a href="/working-mode">Working Mode</a> and the
+  <a href="/faq">FAQ</a>. The original charter is included below for historical
+  context.</strong></p>
```
Comment (domenic) [inline src/charter]: Maybe "for the historical record"

**Comment by domenic** [comment]
I wonder if we should go for something more like the annoying-warning on commit snapshots, or graying out the text, or something...

**Comment by annevk** [comment]
Made it a tad stronger. I didn't want to go all overboard with stylesheets since that seems like quite a bit of work to get right.

File: src/charter [hunk 2]
```diff
@@ -16,16 +16,15 @@
    <li><a href="https://spec.whatwg.org/">Standards</a></li>
    <li><a href="/working-mode">Working Mode</a></li>
    <li><a href="/code-of-conduct">Code of Conduct</a></li>
-   <li class="this"><strong>Charter</strong></li>
+   <li><a href="/faq">FAQ</a></li>
   </ul>
 
-  <h2>Web Hypertext Application Technology Working Group Charter</h2>
+  <h2>Charter</h2>
```
Comment (domenic) [inline src/charter]: Charter (Historical) ?

**Pull request merged**